<launch>
    <arg name="i2c_path"/>
    <arg name="i2c_channel" default="/dev/i2c-2"/>

    <node name="baybot_base" pkg="baybot_base" type="baybot_base_node" output="screen" >
        <rosparam subst_value="true">
            i2c/path: $(arg i2c_path)
            i2c/channel: $(arg i2c_channel)
            control_hz: 1.0
        </rosparam>
    </node>

    <!-- Differential controller parameters -->
    <include file="$(find baybot_control)/launch/control.launch" />

</launch>
